src/renderer/graph/ChartController.tsx

```diff
@@ -4,29 +4,40 @@ import * as d3 from 'd3';
 export const GRAPH_START = 10;
 export const GRAPH_END = 25000;
 
-export interface ChartDataPoint {
-  x: number;
-  y: number;
-}
-
-export interface MarginLike {
+export interface IMarginLike {
   left: number;
   top: number;
   right: number;
   bottom: number;
 }
 
-export interface ChartData {
+export interface IChartPointData {
+  x: number;
+  y: number;
+}
+
+export interface IChartLineDataPointsById {
+  [id: string]: IChartPointData[];
+}
+
+export interface IChartLineData {
```

Comment:
Since we're only using this inside ChartController.tsx, I think it would be better not to export this interface.